test/fixtures/incorrect.js

```diff
@@ -237,6 +237,20 @@ class NoUnderscoreDangle {
 
 noop(new NoUnderscoreDangle());
 
+// `no-unused-expressions`
+const bin = true;
+const biz = val => val;
+
+if (bin) {
+  0;
+}
+
+0;
+
+biz(0), {}
```

Comment:
Missing `;`

Also, sequences aren't allowed ([`no-sequences`](https://eslint.org/docs/rules/no-sequences#top)), so why test this case in this rule?

Reply:
If you keep this test, you can use `noop` instead of `baz`.